lib/internal/modules/cjs/loader.js

```diff
@@ -1072,6 +1072,7 @@ Module._initPaths = function() {
     homeDir = safeGetenv('HOME');
     nodePath = safeGetenv('NODE_PATH');
   }
+  const relativePaths = process.config.variables.node_relative_path;
```

Comment:
`process.config.variables` is mutable by applications, i.e., `process.config.variables = null` would totally break the module loader.

Grep for `internalBinding('config')` in the lib/ directory to see how to obtain the configuration.

Reply:
I have a babies comming in the next few week, feel free to patch